file_paths.md

```diff
@@ -204,6 +217,20 @@ user or group.
             run                               # :rundir                      same
                 agent.pid                     # :pidfile
 
+    C:\ProgramData\PuppetLabs\pxp-agent *
+        etc *
+            pxp-agent.cfg                     # pxp-agent configuration file
+            modules.d *                       # directory where pxp-agent modules are stored
+                puppet.cfg *                  # ../../var/modules/puppet configuration
+        var *
+            log
+                pxp-agent.log                 # enabled by default
+            modules *
+                puppet.bat *                  # executes Puppet and provides results of Puppet runs
```

Comment:
Perhaps it would be better to first understand the purpose of what you want to do to be able to suggest the most relevant course for addressing it on Windows. @parisiale :)